src/PetConstructor.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable func-names */
```

Comment:
Do you need this rule exception here? Always better to avoid using these if at all possible